Azure-Samples/jp-azureopenai-samples
# Add a GitHub Action for static analysis

## Purpose

* To maintain this repo effectively, I added a GitHub Action for static analysis check

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[x] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
poetry install
```

* Test the code

```
poetry run ruff check .
```

## What to Check
Verify that the following are valid
* Return code of `poetry run ruff check .` is 0

## Other Information
